### PR TITLE
additional custom form tests

### PR DESCRIPTION
**Pull request by sinisaos**
Related to #303

### CODE REVIEW

File: e2e/test_codegen.py [hunk 1]
```diff
@@ -60,6 +60,57 @@ def test_row_listing_filter(playwright: Playwright, dev_server) -> None:
...
+    page.get_by_role("spinbutton", name="Year").fill("2023")
+    page.locator('input[name="date_field"]').fill("2023-04-27")
+    page.locator("span").filter(has_text="27").nth(1).click()
+    page.locator('input[name="datetime_field"]').click()
+    page.get_by_role("spinbutton", name="Year").click()
+    page.get_by_role("spinbutton", name="Year").fill("2023")
+    page.locator("span").filter(has_text="27").nth(3).click()
+    page.get_by_role("spinbutton", name="Hour").press("Enter")
+    page.locator(".segment > select").first.select_option("1")
+    page.get_by_role("button", name="Submit").click()
```
Comment (dantownsend) [inline e2e/test_codegen.py:105]: After submitting the form it would be good to see that it was successful.

**Review by dantownsend — COMMENTED** [review]
Looks good - just a couple of comments

File: admin_ui/src/components/InputField.vue [hunk 2]
```diff
@@ -54,6 +54,7 @@
                 -->
                 <flat-pickr
                     v-bind:config="{
+                        allowInput: true,
```
[the remark this answers is not in the document]
Comment (sinisaos) [reply admin_ui/src/components/InputField.vue:57]: Without that line of code, the field is not filled in the Playwright test (the log reports that it is not an editable field). I didn't know how to solve it but to enable `allowInput: true`. If you know another solution, fill free to change this.

File: e2e/test_codegen.py (hunk 1 above)
Comment (sinisaos) [reply e2e/test_codegen.py:105]: I do it tomorrow.

**Comment by sinisaos** [comment]
I closed this PR because I find testing the Flatpickr widget with the Playwright test problematic.

**Pull request closed without merging**